docs/how-it-works/hpo/explain-hpo.rst

```diff
@@ -110,17 +112,49 @@ How a Ray Cluster Operates:
 #.    **Task Scheduling**: Ray automatically schedules trials across nodes based on available resources.
 #.    **Shared State**: Nodes share data such as checkpoints and trial results via a central storage path.
 
-We launch a ray cluster using a dedicated slurm job script. You may refer to `this script <https://github.com/interTwin-eu/itwinai/blob/main/tutorials/hpo-workflows/slurm_hpo.sh>`_ It should be suitable for almost any 
-time you wish to run an itwinai pipeline with Ray, the only thing you may have to change is the ``#SBATCH`` directives to set the proper resource requirements. 
+We launch a ray cluster using a dedicated slurm job script. You may refer to `this script <https://github.com/interTwin-eu/itwinai/blob/main/tutorials/hpo-workflows/slurm_hpo.sh>`_.
+It should be suitable for almost any 
+time you wish to run an itwinai pipeline with Ray, the only thing you may have to change is the ``#SBATCH`` directives to set the proper resource requirements.
+We use this script to launch both of our HPO integrations, changing only the final command, depending on which script we want to execute once our ray cluster is set up.
 Also refer to the `ray documentation <https://docs.ray.io/en/latest/cluster/vms/user-guides/community/slurm.html>`_ 
 on this topic, if you want to learn more about how to launch a ray cluster with slurm.
 
 
-How Distributed Training Works with the RayTorchTrainer
---------------------------------------------------------
+How to Run Your Pipeline with Ray Tune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The easiest way to start running HPO with itwinai is to use our template to wrap a 
+pipeline in a simple function to pass it to a Ray Tune ``Tuner``. This method is suitable for users who want a quick, lightweight setup -
+if you already have an itwinai trainer and pipeline, setting up this integration should not take you
+much more than ten minutes. This method uses only Ray Tune for trial distribution and hyperparameter sampling, 
```

Comment:
Not a big deal, but I think I would omit the direct mention of "ten minutes" here and rather say something more generic like "only includes a few short steps" or something along those lines.